ocrd_validators/ocrd_validators/page_validator.py

```diff
@@ -146,39 +293,29 @@ def validate(filename=None, ocrd_page=None, ocrd_file=None, strictness='strict',
             ocrd_file (OcrdFile): OcrdFile instance wrapping OcrdPage
             strictness (string): 'strict', 'lax', 'fix' or 'off'
             strategy (string): Currently only 'index1'
+            check_baseline (bool): whether Baseline must be fully within TextLine/Coords
+            check_coords (bool): whether *Region/TextLine/Word/Glyph must each be fully
+                                 contained within Border/*Region/TextLine/Word, resp.
 
         Returns:
             report (:class:`ValidationReport`) Report on the validity
         """
         if ocrd_page:
-            validator = PageValidator(ocrd_page, strictness, strategy)
+            page = ocrd_page
+            file_id = ocrd_page.get_pcGtsId()
```

Comment:
Do we require PAGE pcGtsId and mets:file ID to match? @wrznr @tboenig If we do not we should, right?

Reply:
Not sure. Might complicate the processing of existing PAGE.

Reply:
I'm also inclined to say we should, because it could help make logging output more usable. But I agree we also should not rely on that identity. And keep non-empty pcGtsId on import.

Anyway, here it's just "anything is better than nothing".

Reply:
Yeah, this is not related to this PR but we might want to check it in the workspace validation and at least notice users.

Reply:
https://github.com/OCR-D/core/issues/378